### PR TITLE
Hand setters getter

### PR DESCRIPTION
**Pull request by Mchan2003**

### CODE REVIEW

File: Arm/Hand/hand.hpp
Comment (naterpotatoers) [inline Arm/Hand/hand.hpp]:
```suggestion
  Hand(sjsu::arm::WristJoint & wrist) : wrist_(wrist) {}
```
Try this fix and it should fix errors

**Review by naterpotatoers — APPROVED** [review]

**Comment by naterpotatoers** [comment]
Do not merge until this compiles*** I just approved it out of courtesy since it should be an easy fix

File: Arm/Hand/hand.hpp
Comment (Mchan2003) [reply Arm/Hand/hand.hpp]: Hi, I tried to compile the code with your fix but it keeps giving me this error: 

` error: uninitialized reference member in 'class sjsu::RmdX&' [-fpermissive]
   28 | Hand(sjsu::arm::WristJoint & wrist) : wrist_(wrist) {}
   |      ^~~~`

Is the error referring to Hand being uninitialized or is there a reference member in the RmdX class being uninitialized?

Comment (Mchan2003) [reply Arm/Hand/hand.hpp]: Nevermind. I think I solved the error.

Comment (naterpotatoers) [reply Arm/Hand/hand.hpp]: sorry didn't see this earlier

File: Arm/Hand/hand.hpp
Comment (naterpotatoers) [inline Arm/Hand/hand.hpp]: Leave the trailing underscore off. Here's the [Google style guide reference](https://google.github.io/styleguide/cppguide.html#Variable_Names)

File: Arm/Hand/hand.hpp
Comment (naterpotatoers) [inline Arm/Hand/hand.hpp]: We need to pass by reference here still 
```suggestion
  Hand(sjsu::arm::WristJoint & wrist) : wrist_(wrist) {}
```

File: Arm/Hand/hand.hpp
Comment (naterpotatoers) [inline Arm/Hand/hand.hpp:102]: Since we've implemented the Finger.hpp class these actually will go away and be handled in the Finger.hpp class now

File: Arm/Hand/hand.hpp
Comment (naterpotatoers) [inline Arm/Hand/hand.hpp:128]: As mentioned before, these will be replaced by the finger.hpp class which should be public member variables of this class

**Review by naterpotatoers — CHANGES_REQUESTED** [review]
Honestly with the addition of the finger.hpp class we can probably just close out this PR without merging it since we will instead be calling the Finger.hpp class within this class for moving the fingers etc. Sorry about that, probably should have spent more time brainstorming before implementing

**Pull request closed without merging**
